src/userCache.ts

```diff
@@ -11,19 +11,19 @@ export type User = {
   teamMembership: string[];
   manager: string;
   permissions: string[];
-};
+}
 
-type Cache = {
+interface Cache {
   [email: string]: {
     expiration: number;
     user: User;
   };
-};
+}
 
 class UserCache {
   cache: Cache = {};
 
-  set(user: User, lifespan: number) {
+  set(user: User, lifespan: number): void {
```

Comment:
Is there a reason for having these explicit?

Reply:
I couldn't disable that from the eslint rule, seems like it doesn't allow to keep `: void` as optional.